spec/javascripts/controllers/alerts_center/alerts_list_controller_spec.js

```diff
@@ -0,0 +1,243 @@
+describe('alertsListController', function() {
+  var $scope, $controller, alertsCenterService;
+  var adminResponse, operatorResponse, existingUsersResponse, providersResponse, tagsResponse, alertsResponse;
+
+  beforeEach(module('alertsCenter'));
```

Comment:
probably not what you want (unless `module` really returns a function that `beforeEach` should run)